### PR TITLE
Fix CEV-Shielding mapping and remove parsing methods

### PR DESCRIPTION
**Pull request by maysakanoni**
**What?**
- Add some logic in mapping layer which maps Shielding to CEV in front and and CEV to Shielding in the request object to the api
- Added some functions to allow the gateway to manually map response and request object in the postToUrl/patchToUrl and putToUrl functions
- removed `parseCamelToPascalCase` and `parsePascalToCamelCase`  from postToUrl/patchToUrl and putToUrl
- correct typo in accessing postcode field from PostCode to Postcode as it is return in from the api
**Why?**
- Consistency in api layer for shielding cases and call handlers use CEV

### CODE REVIEW

File: src/gateways/help-request-call.js [hunk 1]
```diff
@@ -1,7 +1,17 @@
 import { DefaultGateway } from '../gateways/default-gateway';
 
+const ToPostHelpRequestCall = (hr) => {
+  const callType = (hr.callType == 'CEV') ? 'Shielding': hr.callType
+  return JSON.stringify({
+      CallType: callType,
+			CallDirection: hr.callDirection,
```
Comment (antonyoneill) [inline src/gateways/help-request-call.js:7]: Some surplus whitespace here

File: src/gateways/help-request-call.js [hunk 2]
```diff
@@ -1,7 +1,17 @@
 import { DefaultGateway } from '../gateways/default-gateway';
 
+const ToPostHelpRequestCall = (hr) => {
+  const callType = (hr.callType == 'CEV') ? 'Shielding': hr.callType
```
Comment (antonyoneill) [inline src/gateways/help-request-call.js:4]: I created a new constants file recently, can you move these two words (and other usages of it) to use that now please?

See 
https://github.com/LBHackney-IT/coronavirus-here-to-help-frontend/blob/7074b0269cb4a9a937716230153ce85963a94658/src/helpers/constants.js#L1


https://github.com/LBHackney-IT/coronavirus-here-to-help-frontend/blob/7074b0269cb4a9a937716230153ce85963a94658/src/pages/reassign-call/%5BrequestId%5D/index.js#L8
https://github.com/LBHackney-IT/coronavirus-here-to-help-frontend/blob/7074b0269cb4a9a937716230153ce85963a94658/src/pages/reassign-call/%5BrequestId%5D/index.js#L37

File: src/gateways/help-request.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 import { DefaultGateway } from '../gateways/default-gateway';
 
-const ToHelpRequest = (hr) => {
+const ToHelpRequestDomain = (hr) => {
+    const helpNeeded = (hr.HelpNeeded == 'Shielding')? 'CEV' : hr.HelpNeeded
```
Comment (antonyoneill) [inline src/gateways/help-request.js:4]: might also be a good idea to do this inline on line 15 (and in the other places you do this)_

**Review by antonyoneill — APPROVED** [review]
Just some minor changes please @maysakanoni

File: src/gateways/help-request-call.js (hunk 2 above)
Comment (maysakanoni) [reply src/gateways/help-request-call.js:4]: nice, will do!

File: src/gateways/help-request-call.js (hunk 1 above)
Comment (maysakanoni) [reply src/gateways/help-request-call.js:7]: I cant see this in my editor 🤯

**Pull request merged**